### PR TITLE
Added license file [skip ci]

### PR DESCRIPTION
**Pull request by pjhampton**
References [#5]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,3 +247,7 @@ Of course, you can also check the exception message using regular assertions:
 $this->invoking->format(__FUNCTION__)->on($formatter)->shouldThrow->hasMessage('...');
 $this->invoking->format(__FUNCTION__)->on($formatter)->throws('invalidArgumentException')->hasMessage('...');
 ```
+
+## Licence
+
+bdd-extension is released under the MIT License. See the bundled LICENSE file for detail.
```
Comment (jmaitrehenry) [inline README.md]: Should be BSD-3-Clause, not MIT :)

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Frédéric Hardy nor the names of its contributors
```
Comment (jmaitrehenry) [inline LICENSE]: s/Frédéric Hardy/atoum contributors

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      may be used to endorse or promote products derived from this software
+      without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY FRÉDÉRIC HARDY AND CONTRIBUTORS ``AS IS'' AND ANY
```
Comment (jmaitrehenry) [inline LICENSE]: s/Frédéric Hardy/atoum contributors

**Comment by pjhampton** [comment]
Thanks @jmaitrehenry!! 💙  💛  ❤️ 
Please review @jubianchi

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of atoum contributors nor the names of its contributors
+      may be used to endorse or promote products derived from this software
+      without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY ATOUM CONTRIBUTORS AND CONTRIBUTORS ``AS IS'' AND ANY
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL FRÉDÉRIC HARDY AND CONTRIBUTORS BE LIABLE FOR ANY
```
Comment (jmaitrehenry) [inline LICENSE]: You miss one :)

**Comment by jubianchi** [comment]
Thanks @pjhampton for your contribution and @jmaitrehenry for the review! ❤️

**Pull request merged**